doc/rst/language/spec/domain-maps.rst

```diff
@@ -2,88 +2,77 @@
 
 .. _Chapter-Domain_Maps:
 
-===========
-Domain Maps
-===========
+=============
+Distributions
+=============
 
-A domain map specifies the implementation of the domains and arrays that
+A distribution, also called a *domain map*,
```

Comment:
Should we even mention *domain map* at all here?

Reply:
While it may be odd to introduce a term and never use it again, I leave "domain map" here to serve as the search target for people who encounter this term in our past papers, tutorials, etc.